src/screens/Game.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import Blocks from '../components/Blocks';
+import DiceRoller from '../components/DiceRoller';
+import HoverCard from '../components/HoverCard';
+import TradeBox from '../components/TradeBox';
+const gameBlocks = require('./../data/gameBlocks.json');
+function Game(props) {
+  const [isHovering, setIsHoverering] = useState(false);
+  const [hoverIndex, setHoverIndex] = useState(0);
+  const [money1, setMoney1] = useState(1500);
```

Comment:
keep amount `1500` in a constant variable named `INITIAL_WALLET_AMOUNT` or something similar at the top of your file.